src/bun.js/http.exports.js

```diff
@@ -951,10 +951,10 @@ export class ClientRequest extends OutgoingMessage {
   _final(callback) {
     this.#finished = true;
     this[kAbortController] = new AbortController();
```

Comment:
Since fetch supports `signal` now, we shouldn't need this anymore

Reply:
oh this is for timeouts

I see

yeah we do need this

Reply:
Yeah this is what Ciro just added